library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -60,4 +60,12 @@ object Predef:
     inline def ne(inline y: AnyRef | Null): Boolean =
       !(x eq y)
 
+  extension [T](it: java.lang.Iterable[T])
+    def foreach(f: java.util.function.Consumer[T]): Unit =
+      it.forEach(f)
+
+  extension [T](s: java.util.stream.Stream[T])
```

Comment:
Should we also support `IntStream` and the other primitive streams?

Reply:
It we could support `BaseStream` ([javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/stream/BaseStream.html)), which is the common interface above `Stream`, `IntStream` and all, it would be even better :)

Reply:
I think it would be good to also support `IntStream` and the other primitive streams.